sormas-backend/src/main/java/de/symeda/sormas/backend/event/EventParticipant.java

```diff
@@ -77,6 +78,7 @@
 	private SormasToSormasOriginInfo sormasToSormasOriginInfo;
 	private List<SormasToSormasShareInfo> sormasToSormasShares = new ArrayList<>(0);
 	private VaccinationStatus vaccinationStatus;
+	private boolean deleted;
```

Comment:
please remove "deleted" field from EventParticipant. This field already exists on DeletableAdo which is extended by this entity class.